React/CxxBridge/JSCExecutorFactory.h

```diff
@@ -15,6 +15,10 @@ namespace react {
 
 class JSCExecutorFactory : public JSExecutorFactory {
 public:
+  explicit JSCExecutorFactory(
```

Comment:
This change is needed to make the [RNTester turbo module](https://github.com/facebook/react-native/blob/421ffb05aeb7ccf7013425bdac6b14372325bb26/RNTester/RNTester/AppDelegate.mm#L137) example work.